packages/rendermime/src/outputmodel.ts

```diff
@@ -82,75 +163,68 @@ export class OutputModel implements IOutputModel {
    * Construct a new output model.
    */
   constructor(options: IOutputModel.IOptions) {
```

Comment:
This is the one low-level model I have kept around. This is because it is passed to the rendermime, and ultimately used in rendermime extensions. I've instead tried to make it stateless, cheap to create and dispose, and not much more than a view onto a record.